source/features/bugs-tab.tsx

```diff
@@ -24,16 +24,40 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const cacheKey = (): string => 'bugs-label' + ':' + getRepo()!.nameWithOwner;
 const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+	if (!await cache.get(cacheKey())) {
```

Comment:
Not sure if you missed this line

Reply:
What this line does is cause 2 API calls for every single repo and that's exactly why that issue was closed the first time it was asked.

Reply:
> What this line does is cause 2 API calls for every single repo

Why?